scripts/collector/sqlserver/sql/objectList.sql

```diff
@@ -52,12 +52,12 @@ BEGIN
     , type 
     , type_desc
     , count(*) AS object_count
-    , SUM(LinesOfCode) AS lines_of_code 
+    , ISNULL(SUM(LinesOfCode),0) AS lines_of_code
     FROM 
     (
         SELECT
         s.name as schema_name,
-        TRIM(o.type) as type, 
+        RTRIM(LTRIM(o.type)) as type, 
```

Comment:
```suggestion
        TRIM(o.type) as type, 
```
Can this be simplified to a `TRIM`?

Reply:
TRIM isn't there in versions older than SQL 2018.  So you have to do a LTRIM+RTRIM for backwards compatibility.